CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+
+#Changelog
+
+## 2020-07-06
+Initial alert release (weekly w_2020_27).
```

Comment:
What exactly does the version refer to? Was all the processing performed with this version, including alert serialization? Presumably templates etc were generated with some other version?

I don't think we need an exhaustive list of versions of everything sufficient for reproduction, but even just saying “containing data processed with weekly...” seems like it'd be a bit more explicit.

Reply:
As you said, we're not being fully replaceable here (at least not yet).  I'll go with "Software used to process the data is included in weekly w_2020_27" which I think is a reasonable statement of reality.